siliconcompiler/tools/openroad/_apr.py

```
@@ -1258,6 +1260,24 @@ def add_openroad_skipreport(self, report_type: Union[List[str], str],
             clobber: If True, overwrites the existing list of skipped reports.
                      If False, appends to the existing list.
         """
+        if isinstance(report_type, str):
+            patterns = [report_type]
+        else:
+            patterns = list(report_type)
+
+        if any("*" in pattern for pattern in patterns):
+            enum_type = list(NodeType.parse(self.get("var", "skip_reports", field="type")))[0]
```

Ruff (0.15.12)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)